src/axom/quest/tests/quest_intersection_shaper.cpp

```diff
@@ -128,20 +131,15 @@ mfem::GridFunction *newGridFunction(mfem::Mesh *mesh)
 
 void makeTestMesh(sidre::MFEMSidreDataCollection &dc, bool initialMats)
 {
-  int polynomialOrder = 1;
-  double lo[] = {0., 0., -0.25};
-  double hi[] = {1., 1., 0.};
-  int celldims[] = {20, 20, 1};
-  auto mesh =
-    new mfem::Mesh(mfem::Mesh::MakeCartesian3D(celldims[0],
-                                               celldims[1],
-                                               celldims[2],
-                                               mfem::Element::HEXAHEDRON,
-                                               hi[0] - lo[0],
-                                               hi[1] - lo[1],
-                                               hi[2] - lo[2],
-                                               false));
-  mesh->SetCurvature(polynomialOrder);
+  const int polynomialOrder = 1;
+  const auto celldims = primal::NumericArray<int, 3> {20, 20, 1};
+  const auto bbox = primal::BoundingBox<double, 3> {{0., 0., 0.}, {1., 1., .25}};
+
+  auto mesh = quest::util::make_cartesian_mfem_mesh_3D(bbox,
+                                                       celldims,
+                                                       polynomialOrder,
+                                                       false);
```

Comment:
@BradWhitlock -- I had to adjust the bounding box here since the original formulation was not actually offseting the z-coordinate by `-.25`.